ONSdigital/census-rm-sample-loader
# (IC03.1/IC04.1) Generate CE UNIT Addressed ICL Print Files

# Motivation and Context
For a CE U case, error needs to throw if CE_EXPECTED_CAPACITY is not a value of 1 or more

# What has changed
New validators
New tests

# How to test?
Run sample file to check for validation...change value of expected capacity to blank/0/letter etc. and check it fails

# Links
https://trello.com/c/FPqeHTfQ/1046-ic031-i041-generate-ce-unit-addressed-icl-print-files-3
https://collaborate2.ons.gov.uk/confluence/display/SDC/RM+Census+Initial+Contact+Run+Book+-+2021
https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?pageId=34832584
